frontends/web/src/routes/account/account.tsx

```diff
@@ -72,6 +73,15 @@ export function Account({
 
   const account = accounts && accounts.find(acct => acct.code === code);
 
+  const getBitsuranceGuideLink = (): string => {
+    switch (i18n.resolvedLanguage) {
+    case 'de':
+      return 'https://bitbox.swiss/redirect/bitsurance-segwit-migration-guide-de';
+    default:
+      return 'https://bitbox.swiss/redirect/bitsurance-segwit-migration-guide-en';
+    }
+  };
```

Comment:
Do we have any blockers in publishing the guide and making the redirects? I'd prefer to just merge the PR to avoid blocking the next RC. cc @Stadicus

Reply:
ack